includes/definitions/discovery/dell-rpdu.yaml

```diff
@@ -80,3 +80,20 @@ modules:
                         - { descr: normal, graph: 1, value: 2, generic: 0 }
                         - { descr: nearOverload, graph: 1, value: 3, generic: 1 }
                         - { descr: overload, graph: 1, value: 4, generic: 2 }
+        temperature:
+            data:
+                -
+                    oid: rPDUSensorTempStatusTempC
+                    value: rPDUSensorTempStatusTempC
```

Comment:
You can delete these 2 `value:` lines. The `oid` is the same so that will be used.